README.md

```diff
@@ -1,9 +1,11 @@
 ![emscripten logo](media/switch_logo.png)
 
-[![CircleCI](https://circleci.com/gh/emscripten-core/emscripten.svg?style=svg)](https://circleci.com/gh/emscripten-core/emscripten/tree/master)
-
 Main project page: <http://emscripten.org>
 
+GitHub CI status: [![CircleCI](https://circleci.com/gh/emscripten-core/emscripten.svg?style=svg)](https://circleci.com/gh/emscripten-core/emscripten/tree/master)
```

Comment:
"GitHub CI"  is perhaps misleading.  How about just "CI status?

Reply:
I do want to differentiate it from the chromium CI that is just below it now.

Maybe "CircleCI on GitHub"?

Reply:
Is Chromium CI result useful to general audience?

Reply:
@RReverser Less, but it can be useful for users that do `emsdk install tot` and want to know what the latest build there is.